packages/codegen/src/__tests__/codegen.uint.spec.ts

```diff
@@ -110,7 +110,7 @@ describe('code generation', () => {
         'LimitList',
       ];
 
-      const typeNames = await gen.generateTypes(basicSnapshot);
+      const typeNames = await gen.generateTypes(basicSnapshot, []);
```

Comment:
change could be removed if type is optional, see other comment. Same for other places.